test_multi/a.js

```diff
@@ -0,0 +1,6 @@
+import b from './b.js';
```

Comment:
These test files should not be included.

Reply:
Oh, whoops